roles/memcached/tasks/main.yml

```diff
@@ -24,4 +24,4 @@
   notify: restart memcached
 
 - name: Ensure Memcached is started and set to run on startup.
-  service: name=memcached state=started enabled=yes
+  ansible.builtin.service: name=memcached state=started enabled=yes
```

Comment:
Same thing here. Move the param values (name, state, enabled) onto separate lines, all indented 2 spaces from the module name.

Reply:
Will do - thanks!